### PR TITLE
[ADD] Adding extended localization, Vauxoo/mexico

### PR DESCRIPTION
**Pull request by hugho-ad**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+	branch = 14.0
+[submodule "hr"]
+	path = hr
+	url = git@github.com:Vauxoo/hr.git
+	branch = 14.0
+[submodule "hr-advanced"]
+	path = hr-advanced
+	url = git@git.vauxoo.com:vauxoo/hr-advanced.git
+	branch = 14.0
+[submodule "server-tools"]
```
Comment (luisg123v) [inline .gitmodules]: Is server-tools really required?

I ask because, if it's only to install `company_country`, that module is deprecated since 13.0

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (hugho-ad) [reply .gitmodules]: I added it because Vx/mexico has it 
https://git.vauxoo.com/vauxoo/mexico/-/blob/14.0/oca_dependencies.txt#L2

Comment (hugho-ad) [reply .gitmodules]: you right, it's not need it

**Review by luisg123v — APPROVED** [review]
LGTM :+1:


FYI I just submitted [this MR](https://git.vauxoo.com/vauxoo/mexico/-/merge_requests/2867) to fix mexico

**Pull request merged**